ui/menus.md

```diff
@@ -15,13 +15,13 @@ The main menu styling supports 3 level, that is 2 levels of menu section. Menus
 Menu::Section.new(:clo, N_("Clouds"), 'fa fa-plus fa-2x', [
 ```
 
-Menu sections looks like the example above. It consists of: 
- * Identifier, 
- * text, 
- * icon and 
+Menu sections looks like the example above. It consists of:
+ * Identifier,
+ * text,
+ * icon and
  * items.
 
-Identifier is the same as the [RBAC feature](rbac_features.md) that is checked when desicison is make whether the particular section is displayed.
+Identifier is the same as the [RBAC feature](rbac_features.md) that is checked when decision is make whether the particular section is displayed.
 
```

Comment:
`when decision is make whether the particular section is displayed` ->
`when a decision is made as to whether or not the particular section is displayed`